### PR TITLE
Add unistore accessor implementation

### PR DESCRIPTION
**Pull request by apfelbox**

### CODE REVIEW

File: tests/cases/unistore/get.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                // renamed
+                accessor: get`prop1:address.city,prop2:age`,
+                expected: {
+                    prop1: "New York",
+                    prop2: 25,
+                },
+            },
+            {
+                // spaces don't matter (aliased)
+                accessor: get`prop1   :    address   .   city  ,   prop2  :   age`,
```
Comment (keichinger) [inline tests/cases/unistore/get.js]: Is having a `\n` a viable use-case so we can properly format complex retrievals like regular objects/arrays?

Comment (apfelbox) [reply tests/cases/unistore/get.js]: This is already covered, as the whitespace stripping includes newlines as well:

2018-05-15 at 12 09

I have added an explicit test case though. Also trailing commas are now allowed

Comment (keichinger) [reply tests/cases/unistore/get.js]: Great!

**Comment by keichinger** [comment]
Thanks @apfelbox  🎉

**Pull request merged**